### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ivankovbohdan**
- [DEMO LINK](https://ivankovbohdan.github.io/layout_stars/)
- [TEST REPORT LINK](https://ivankovbohdan.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,11 +3,58 @@
...
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star stars__star--active"></div>
```
Comment (anstsot) [inline src/index.html:21]: do not use `active` modifier class for star. rely only on parent `stars--N` class

**Review by anstsot — CHANGES_REQUESTED** [review]
Good work, but that `active` class should not be used.
Instead, you should rely only on `stars--N` class. You can use `:nth-child` selector - [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child).

**Review by YegorKorenkov — CHANGES_REQUESTED** [review]
Hi, you probably didn’t really understand what your mistake is, you don’t need to set any additional classes for stars, except for stars__star. You have to make the stars active depending on the parent container's modifier class stars--1(should be 1 gold star), stars--2(should be 2 active stars)

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+.stars--3 .stars__star:nth-child(n+4) {
+  background-image: url("images/star.svg");
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url("images/star.svg");
+}
+
+.stars--5 .stars__star:nth-child(n+6) {
```
Comment (anstsot) [inline src/style.css:41]: you can merge the styles
```suggestion
.stars--0 .stars__star:nth-child(n+1),
.stars--1 .stars__star:nth-child(n+2),
.stars--2 .stars__star:nth-child(n+3),
.stars--3 .stars__star:nth-child(n+4),
.stars--4 .stars__star:nth-child(n+5),
.stars--5 .stars__star:nth-child(n+6) {
```

**Review by anstsot — CHANGES_REQUESTED** [review]
Almost done, let's add the last fix

**Review by anstsot — APPROVED** [review]
Good work, approving:sparkles:

**Pull request closed without merging**